src/config/configuration.ts

```diff
@@ -60,12 +61,12 @@ const configuration = () => {
     proposalTypes: process.env.PROPOSAL_TYPES_FILE || "proposalTypes.json",
     metricsConfig: process.env.METRICS_CONFIG_FILE || "metricsConfig.json",
   };
-  Object.keys(jsonConfigFileList).forEach((key) => {
-    const filePath = jsonConfigFileList[key];
+  Object.keys(yamlConfigFileList).forEach((key) => {
+    const filePath = yamlConfigFileList[key];
     if (fs.existsSync(filePath)) {
       const data = fs.readFileSync(filePath, "utf8");
       try {
-        jsonConfigMap[key] = JSON.parse(data);
+        jsonConfigMap[key] = load(data, { filename: filePath });
```

Comment:
Can you confirm that we can load both json and yaml files?

Reply:
Yes.